### DISCUSSION TITLE
MemberCacheFlags needed for single-server bot?

### DISCUSSION THREAD

**Discussion by nerrixde** [Rapptz/discord.py#5881] [Q&A] [answered]
I have a single-server bot which needs to reliably call `on_member_update`, `on_member_join` and similar events. Do I need to deal with `MemberCacheFlags`? The single server the bot is on has about 8k members.
ofc I'll enable intents (& specify them in code)

**Comment by Rapptz** [comment, marked as answer]
In order for `on_member_update` to be called, there needs to be some form of member cache so it's best to enable it.
